.github/workflows/ci.yaml

```diff
@@ -23,11 +23,31 @@ jobs:
           cp test/buf.bash tmp/bin/buf
           chmod +x tmp/bin/buf
           echo "${PWD}/tmp/bin" >> "${GITHUB_PATH}"
-      - name: run action
+      - name: run on branch disable draft
+        if: github.ref_type == 'branch' && github.ref != 'refs/heads/main'
         uses: ./
         with:
           buf_token: fake-buf-token
           input: path/to/input
         env:
           WANT_BUF_TOKEN: fake-buf-token
-          WANT_ARGS: push --tag ${{ github.sha }} path/to/input
+          WANT_ARGS: push path/to/input --tag ${{ github.sha }}
```

Comment:
Is this `WANT_` stuff somehow related to the tests being run from `ci.yaml`? I'm a bit confused about what is going on here.

Reply:
yes, these `WANT_` stuffs are basically the expected test result